### ISSUE TITLE
'aec configure example' fails

### ISSUE THREAD

**Issue by koterpillar** [seek-oss/aec#304]
Installing:

```
» pipx install aec-cli                                               
  installed package aec-cli 0.9.0, Python 3.9.5
  These apps are now globally available
    - aec
done! ✨ 🌟 ✨
```

Running the `configure example`:

```
» aec configure example
Traceback (most recent call last):
  File "/<HOME>/.local/bin/aec", line 8, in <module>
    sys.exit(main())
  File "/<HOME>/.local/pipx/venvs/aec-cli/lib/python3.9/site-packages/aec/main.py", line 129, in main
    result = cli.dispatch(build_parser(), args)
  File "/<HOME>/.local/pipx/venvs/aec-cli/lib/python3.9/site-packages/aec/util/cli.py", line 87, in dispatch
    return call_me(**vars(pargs))
  File "/<HOME>/.local/pipx/venvs/aec-cli/lib/python3.9/site-packages/aec/util/configure.py", line 11, in example
    with resources.path("aec", "config-example") as example_path:
  File "/usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/contextlib.py", line 117, in __enter__
    return next(self.gen)
  File "/usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/importlib/resources.py", line 175, in _path_from_reader
    opener_reader = reader.open_resource(norm_resource)
  File "<frozen importlib._bootstrap_external>", line 1060, in open_resource
IsADirectoryError: [Errno 21] Is a directory: '/<HOME>/.local/pipx/venvs/aec-cli/lib/python3.9/site-packages/aec/config-example'

**Comment by tekumara** [comment]
Thanks for the bug report @koterpillar! Looks like this is broken in Python 3.9 because of changes to importlib resources, but older versions are OK.

**Comment by tekumara** [comment]
Fixed in #308
